eregs_extensions/atf_regparser/tests/preprocessor_tests.py

```diff
@@ -0,0 +1,23 @@
+# vim: set encoding=utf-8
+from unittest import TestCase
+
+from tests.xml_builder import XMLBuilderMixin
```

Comment:
Bleh. I suppose this means we should move the `tests` module from `regparser` into `regparser.tests`?